vocab/properties.json

```diff
@@ -1838,7 +1980,9 @@
     },
     "description": "Aspects or standards on which a judgement or decision is based.",
     "label": "Criteria",
+    "labelPlural": "Criterias",
```

Comment:
"criteria" is already the plural of "criteron". "Criterias" is not correct.

Reply:
you are right, we should then correct the property to criterion